components/item/AwesomeSwiper.vue

```diff
@@ -10,12 +10,23 @@
       v-for="(item, index) in displayableMedia"
       :key="index"
     >
+      <div
+        v-if="singleMediaResource"
+        class="container h-100"
+      >
+        <MediaCard
+          :europeana-identifier="item.europeanaIdentifier"
+          :media="item"
+        />
+      </div>
       <MediaCard
+        v-else
         :europeana-identifier="item.europeanaIdentifier"
         :media="item"
       />
     </swiper-slide>
     <div
+      v-if="swipingEnabled"
```

Comment:
Is this perhaps a leftover? I don't see any other mention of `swipingEnabled` in this file and there is a "Property or method "swipingEnabled" is not defined on the instance but referenced during render..." warning.